DSCSTools/EXPA.cpp

```diff
@@ -264,11 +274,11 @@ namespace dscstools {
 
 			boost::property_tree::ptree format = getStructureFile(source);
 
-			std::size_t numTables = format.size();
+			std::size_t numTables = 0;// format.size();
 
 			// write EXPA Header
 			output.write("EXPA", 4);
-			output.write(reinterpret_cast<char*>(&numTables), 4);
+			output.write(reinterpret_cast<char*>(&numTables), 4); // Going to overwrite this later
```

Comment:
remove it if you're writing it later

Reply:
Would you prefer to seek ahead by 4 bytes in this line or explicitly write 0 (current behaviour)?